docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -130,14 +130,14 @@ The connector supports pushdown for a number of operations:
 * :func:`min`
 * :func:`sum`
 
-Limitations
+.. _clickhouse-sql-support:
+
+SQL support
 -----------
 
-The following SQL statements aren't  supported:
+The connector provides read access to data and metadata in
```

Comment:
This connector (the seems like it is disambiguating from a component that is not a connector)

Reply:
I agree, but right now "the connector" seems to be standard nomenclature across this and other connector docs. May be worth follow up later.

Reply:
Clickhouse connector supports writes too. Not sure this statement is factually correct anymore.

Reply:
Updated. Are there any other supported write statements not listed here?

Reply:
INSERT is missing. DELETE and UPDATE aren't supported.

Reply:
Added. Also broke out the schema statements to be more granular as discussed in https://github.com/trinodb/trino/pull/8669#discussion_r677568652 if you think that's appropriate here